### PR TITLE
feat(storage/contract): remove abi and bytecode from contract_code

### PR DESCRIPTION
**Pull request by kkovaacs**
Turns out we don't really use these fields anymore. We still need to keep some legacy code around for old migration steps, though, including the `extract_abi_code_hash` function...

Fixes: #905

### CODE REVIEW

**Review by CHr15F0x — APPROVED** [review]
LGTM

File: crates/gateway-types/src/class_hash.rs [hunk 1]
```diff
@@ -48,6 +57,9 @@ fn parse_contract_definition(
 
 /// Sibling functionality to only [`compute_class_hash`], returning also the ABI, and bytecode
 /// parts as json bytes.
+///
+/// NOTE: This function is deprecated. We no longer stora ABI and bytecode in the database,
```
Comment (CHr15F0x) [inline crates/gateway-types/src/class_hash.rs:61]: nit: just a typo

```suggestion
/// NOTE: This function is deprecated. We no longer store ABI and bytecode in the database,
```

**Review by CHr15F0x — APPROVED** [review]
LGTM :+1:

File: crates/gateway-types/src/class_hash.rs (hunk 1 above)
Comment (kkovaacs) [reply crates/gateway-types/src/class_hash.rs]: Fixed, thanks!

**Review by CHr15F0x — APPROVED** [review]

**Pull request merged**